src/components/Artist/Artist.css

```diff
@@ -0,0 +1,47 @@
+.Artist {
```

Comment:
Consider using a consistent order, i.e. always putting height/width first, then margins/padding, etc...whatever feels logical to you.